ara/ui/templates/partials/playbook_results.html

```diff
@@ -174,3 +181,14 @@ <h3>No results found matching your query: try <a href="{% url 'ui:playbook' play
   </div>
 </div>
 <br/>
+<script>
+  $(function () {
+    $('[data-toggle="tooltip"]').tooltip()
+  })
+</script>
+<style>
+  /* Prevent tooltip flickering when using hover */
+  .tooltip {
+    pointer-events: none;
+  }
+</style>
```

Comment:
Could you add the ``<script>`` to host_results.html too ? The tooltip doesn't work there because it's missing.

For the style block, could you move it to [ara.css](https://github.com/ansible-community/ara/blob/master/ara/ui/static/css/ara.css) instead ?

Thanks !